### ISSUE TITLE
Line: Large Distance over IDL causes problem

### ISSUE THREAD

**Issue by Jxck-S** [flopp/py-staticmaps#37]
Mapping a line over the IDL with a significant distance causes the following problem.
![image](https://github.com/flopp/py-staticmaps/assets/20673070/e2c1d0ef-f51d-4bef-ad2d-73247e05843b)

Code used to create, this occurs with Pillow and cairo. SVG not tested
```python
jfk = (40.641766, -73.780968)
hong_kong_airport_coordinates = (22.308046, 113.918480)
first = staticmaps.create_latlng(jfk[0], jfk[1])
sec = staticmaps.create_latlng(hong_kong_airport_coordinates[0], hong_kong_airport_coordinates[1])
line = staticmaps.Line([first, sec])
static_map_context.add_object(line)


image = static_map_context.render_pillow(1920, 1080, attribution=False)
image.save("testproblem"+".png")

```

**Comment by Jxck-S** [comment]
You can see the X/y's being output are not connecting properly 
![image](https://github.com/flopp/py-staticmaps/assets/20673070/1432fba8-500f-431c-aebf-9a8ff910d8c9)

**Comment by Jxck-S** [comment]
Bug Bountied this. [https://x.com/jxck_sweeney/status/1757647752224211074?s=46&t=FtgP2n3lVDy-7UKDHwqlcQ](https://x.com/jxck_sweeney/status/1757647752224211074?s=46&t=FtgP2n3lVDy-7UKDHwqlcQ)

@FailSpy please contact me as it seems you fixed it. JackHSweeney@protonmail.com

**Comment by FailSpy** [comment]
@Jxck-S Joined the Ground Control server as FailSpy on Discord. Will reach out to you there.

**Comment by flopp** [comment]
Fixed by #38 38
